### PR TITLE
Dead Link Fixed

### PR DESCRIPTION
**Pull request by saloni-garg**
Just fixed a dead link - https://www.jaegertracing.io
Signed-off-by: Saloni Garg salonigarg2310@gmail.com

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Please see [CONTRIBUTING.md](CONTRIBUTING.md).
 [project]: https://readthedocs.org/projects/jaeger/
 [ci-img]: https://travis-ci.org/jaegertracing/documentation.svg?branch=master
 [ci]: https://travis-ci.org/jaegertracing/documentation
-[jaeger]: https://jaegertracing.io/
+[jaeger]: http://jaeger.readthedocs.io/en/latest/
```
Comment (yurishkuro) [inline README.md]: the link was correct, just needs to be http, not https

**Comment by saloni-garg** [comment]
Done :+1:

**Comment by pavolloffay** [comment]
@saloni-garg you have to sign commits. It can be done with `git commit -s -m 'msg"`

**Comment by saloni-garg** [comment]
Okay @pavolloffay  I'll take care of it the next time onwards :+1:

**Pull request merged**